src/mapHelper.js

```diff
@@ -13,7 +13,8 @@
 module.exports = {
 	browserFamily: {
 		'Yandex Browser': 'yandex',
-		'Android': 'Android Browser'
+		'Android': 'Android Browser',
+		'Chrome Mobile': 'Android Browser'
```

Comment:
I've confirmed at <https://api.browserstack.com/4/browsers> that indeed even for current Android versions, BrowserStack uses "Android Browser". This is unexpected since hthe "Live" interface has both Chrome/Firefox/Samsung Internet as separate browsers on recent Android devices. Anway, LGTM.